wsi/drm/drmplane.cpp

```diff
@@ -514,10 +513,17 @@ bool DrmPlane::IsSupportedModifier(uint64_t modifier, uint32_t format) {
     const format_mods& obj = formats_modifiers_.at(i);
     if (obj.format == format) {
       std::vector<uint64_t>::const_iterator it;
+#ifdef OS_YUNOS
```

Comment:
hmm not sure why this is marked as resolved. I dont understand why this is YUNOS specific.

Reply:
I'm a bit confused on this also, the auto keyword is newer than std::find... Maybe you just need to "#include <algorithm>" somewhere?

Reply:
yes, after add “#include <algorithm>” can pass the build now. Thanks!